packages/react/package.json

```diff
@@ -85,6 +85,8 @@
     "@popperjs/core": "^2.11.8",
     "classnames": "^2.3.1",
     "dom-helpers": "^5.2.1",
+    "embla-carousel": "^8.6.0",
```

Comment:
Based on the docs, only the embla-carousel-react library is needed. Currently embla-carousel alone is being used to import a type and I'm sure that it should also be exported from embla-carousel-react